src/dune_rules/coq/coq_rules.ml

```diff
@@ -132,6 +132,16 @@ let coqc_native_flags ~sctx ~dir ~theories_deps ~theory_dirs
       ; "-native-compiler"
       ; "ondemand"
       ]
+  | VosOnly ->
+    Command.Args.As
+      [ "-vos"
```

Comment:
Since you add vos in file_flags do you need this here? Can be the same as VoOnly I think.

Reply:
Yeah, the other one (variable `file_flags`) is for `coqdep`.

Reply:
I opted to always call `coqdep` with the `-vos` option, so that we can share the dependency analysis between modes.